akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/Effect.scala

```diff
@@ -79,7 +79,7 @@ object Effect {
    * commands will not be processed by this `unstashAll` effect and have to be unstashed
    * by another `unstashAll`.
    *
-   * @see [[Effect.thenUnstashAll]]
+   * @see [[EffectBuilder.thenUnstashAll]]
```

Comment:
Doesn't that become `EffectBuilder#thenUnstashAll` when it is on an instance rather than static?

When is `.` used and when is `#` used?

Reply:
I think `#` and `.` are equal in such links. IntelliJ seems to have some better support for `.` so I think we should prefer that, but it's not important. I just changed those that Scaladoc complained about.